.github/renovate.json5

```diff
@@ -116,7 +116,7 @@
         "org.slf4j:slf4j-api",
         "org.springframework.boot",
         "org.springframework.boot:spring-boot-dependencies"],
-      "matchUpdateTypes": ["major"],
+      "matchUpdateTypes": ["major", "minor"],
```

Comment:
i'm not sure we want this, it will prevent renovate from updating us from Spring Boot 3.3 to 3.4 in the future

Reply:
no, spring 3 uses https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/dc4330e0a3060bd7d8c4090ad0b8fc4727e68113/settings.gradle.kts#L43-L45

I added a comment explaining this